<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ТСОиК ВИМ-3U-3 программа тестирования ИЖДЯ.00009</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string>Наборы тестов</string>
    </property>
    <widget class="QPushButton" name="b_test_selectall">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выделить всё</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_test_selectnone">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Снять выделение</string>
     </property>
    </widget>
    <widget class="QListWidget" name="l_test_selectlst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>421</width>
       <height>671</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Проверка загрузки U-boot</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Комплексный тест NOR1 и NOR2</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Запись Linux OS в NOR1</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка сигнала перезагрузки</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Запуск Linux</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта TRS</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Запись тестовых скриптов</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка доступа к ПЛИС</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка РПЗУ NAND</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка ЭЗУ nvRAM №1</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка ЭЗУ nvRAM №2</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка РПЗУ I2C EEPROM</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка I2C термодатчиков</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта Ethernet (TEth)</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта Ethernet AFDX1</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта Ethernet AFDX2</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта PCIe_VPX</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта I2C_VPX</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта PCIe_XMC</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта I2C_XMC</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта RS232C_1</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта RS232C_2</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка порта RS485</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сигналы XMC, сквозные дифференциальные. Прохождение сигнала 125 МГц</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. (DS(0)..DS(9)) Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. (DS(15)..DS(22)). Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 27В/Обрыв. (DS(15)..DS(22)). Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. Выходы.  Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 27В/Обрыв. Выходы.  Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК Готовность. 0В/Обрыв. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК Низковольтные. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сигналы XMC, сквозные. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сигналы XMC, служебные. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сигналы VPX, однонаправленные. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сигналы VPX, двунаправленные, входы. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сигналы VPX, двунаправленные, выходы. Срабатывание</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проверка метки времени</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. L1 (DS(0)..DS(9),DS(15)..DS(22)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. L0 (DS(0)..DS(9),DS(15)..DS(22)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 27В/Обрыв. L1 (DS(0)..DS(9),DS(15)..DS(22)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 27В/Обрыв. L0 (DS(0)..DS(9),DS(15)..DS(22)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. L1 (DS(23),DS(24)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 0В/Обрыв. L0 (DS(23),DS(24)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 27В/Обрыв. L1 (DS(23),DS(24)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РК 27В/Обрыв. L0 (DS(23),DS(24)) Проверка характеристик</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QPushButton" name="b_ctrl_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Запустить тесты</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_ctrl_stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Остановить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_ctrl_savereport">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить отчёт в файл</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="pb_ctrl_test_progress">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Отчёт</string>
    </property>
    <widget class="QTextEdit" name="te_report">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>24</y>
       <width>611</width>
       <height>421</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Averia Sans'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>IPMI информация</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ввод информации</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Серийный номер</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC-адрес 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC-адрес 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC-адрес 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>550</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Запись IPMI</string>
    </property>
    <widget class="QPushButton" name="b_write_ipmi">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Записать IPMI</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="ipmi_report">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>70</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="f_quick_write_ipmi">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Вторичная запись</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
